main.py

```diff
@@ -83,13 +100,17 @@ async def on_message(message):
 
 
 @bot.command()
+@commands.has_permissions(administrator=True)
```

Comment:
I think this is my fault but these changes which are from my version should not appear as commits in this PR. Likely a problem with how I have the repo structured. Will look into changing this for the next release.